vnext/Microsoft.ReactNative/Views/Text/TextPropertyChangedParentVisitor.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <Views/TextViewManager.h>
+#include "TextParentVisitor.h"
+
+namespace Microsoft::ReactNative {
+
+enum class PropertyChangeType : std::uint_fast8_t {
+  None = 0,
+  Text = 1 << 0,
+  AddHighlight = 1 << 1,
```

Comment:
for my information - why use the shift operator, rather than just write 0,1,2..? Wouldn't that get cast to the underlying enum type?

Reply:
This is a flags enum.

Reply:
(i.e., supports &, &=, etc.)